pkgs/by-name/co/concord/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  curl,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "concord";
+  version = "2.2.1";
+
+  src = fetchFromGitHub {
+    owner = "Cogmasters";
+    repo = "concord";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-8k/W6007U1/s3vx03i1929a5RKZtpW/jOr4JDwmzwp8=";
+  };
+
+  makeFlags = [ "PREFIX=${placeholder "out"}"];
+
+  buildInputs = [ curl ];
+
```

can you try to split lib archive and headers?
```suggestion
outputs = [ "out" "dev" ];
```